### DISCUSSION TITLE
Consistency with Preconditioned Latents

### DISCUSSION THREAD

**Discussion by scott-vector** [neonbjb/tortoise-tts#147] [Q&A]
I do realize there is a post about consistency when blending pickle files, but I am looking to see if @neonbjb or anyone else knows how to generate consistent (as close to identical) results with a single PTH. I've tried using seeds, and I've messed around with the temp and top_p, but I am not sure if I am going further away from consistency or closer to it.

The results do have extremely similar prosody and pitch, but the placement of the audio and the tempo of the clips don't match up. What I am looking for is a way to feed tortoise a pickle and have the exact or close to exact result, hopefully without sacrificing realism.

**Comment by vdruts** [comment]
Scott anyway you can give a 4dummies explanation on how to create and utilize the latents?

**Comment by vdruts** [comment]
Does using --seed do this? Also is there a way to determine which seed was used when I get an ouput where random seeds were used? Say I'm running an experiment to find the best seed. How do I figure out which seed was used? It does not display in any of the output on the CLI.
